server/data/educationAndWorkPlanClient.ts

```diff
@@ -7,17 +7,23 @@ export default class EducationAndWorkPlanClient {
     return new RestClient('Education and Work Plan API Client', config.apis.educationAndWorkPlan, token)
   }
 
-  async createGoal(createGoalRequest: CreateGoalRequest, token: string): Promise<void> {
-    await EducationAndWorkPlanClient.restClient(token).post({
+  async createGoal(createGoalRequest: CreateGoalRequest, token: string): Promise<unknown> {
```

Comment:
Much like the `curiousClient` above, I'm not sure how this worked before - not sure how a method declaring it returns a `Promise` works when it uses the `await` keyword ... 🤔